doc/config/layers.md

```diff
@@ -105,6 +105,7 @@ Example:
 * **track**: Classify a layer's track direction. This will usually be either ascending or descending and will be displayed within the title of the layer in the Measurement / Sources add modal view if provided.
 * **description**: Point to a markdown file within the metadata folder to provide a layer description.
 * **wrapX**: Wrap the layer across the anti-meridian.
+* **snapshot**: Ability to be downloaded via worldview snapshots (WVS). Default is `true`.
```

Comment:
I think adding a new property to configs where the default value is considered `true` is a bit awkward in JavaScript since an `undefined` value is considered `false`.  It might be preferable to to name the property something like `disableSnapshot` so the default value can be `false`.  This allows a check for matching layers to look like:
```
visibleLayers.filter(({ disableSnapshot }) => !disableSnapshot);
```
instead of the following which hurts my head a little bit to look at:
```
visibleLayers.filter(({ snapshot = true }) => !snapshot);
```